gourmet/keymanager.py

```diff
@@ -135,55 +136,64 @@ def get_key (self,txt, certainty=0.61):
         debug("End get_key",10)
         return k
 
-    def look_for_key (self, txt):
-        """handed a key, return a sorted ALIST of potential keys and
-        scores.
+    def look_for_key(self, txt: str) -> list:
```

Comment:
We know what kind of list it is. It's a list of pairs with `str` and `float`

Reply:
I've updated it